src/Common/MoistThermodynamics/states.jl

```diff
@@ -201,3 +201,35 @@ function LiquidIcePotTempSHumNonEquil_given_pressure(θ_liq_ice::FT, p::FT, q_pt
     e_int = internal_energy(T, q_pt)
     return PhaseNonEquil(e_int, ρ, q_pt)
 end
+
+"""
+    guaranteed_convergence_range(FT)
+
+A range of input arguments to thermodynamic state constructors
+ - `e_int`
+ - `ρ`
+ - `q_tot`
+ - `q_pt`
+ - `T`
+that are guaranteed converge in saturation adjustment.
```

Comment:
```suggestion
that are guaranteed to converge in saturation adjustment.
```
`guaranteed` is pretty strong--maybe stronger than we can really claim here (e.g., when PlanetParameters change this may not hold). But it's ok for now.